src/scripts/HorizontalDynseqTrack.js

```diff
@@ -214,7 +214,12 @@ export default function HDT(HGC, ...args) {
         .range([tileX, tileX + tileWidth]);
       let width = (this._xScale(tileX + tileWidth) - this._xScale(tileX)) / dlen;
 
-      const middle = this.valueScale(offsetValue);
+      if (this.valueScale.domain()[0] > 0) {
```

Comment:
This block is the only material change.

Reply:
Fixed this manually to avoid the prettier changes, thanks for investigating!

Reply:
Published as v0.1.8.